parser.js

```diff
@@ -73,6 +87,15 @@ const metadataRules = {
       ['link[rel="Shortcut Icon"]', node => node.element.getAttribute('href')],
       ['link[rel="mask-icon"]', node => node.element.getAttribute('href')],
     ],
+    scorers: [
+      (node, score) => {
+        const sizes = node.element.getAttribute('sizes');
+
+        if (sizes) {
+          return score + sizes.match(/\d+/g).reduce((a, b) => a * b);
```

Comment:
Oh, but I tried this in the console:

```
>> "any".match(/\d+/g).reduce((a,b) => a*b);
x TypeError: "any".match(...) is null 
```

So you should add some handling `size` attribute values that aren't "NxM"